weaknesses/MASVS-PRIVACY/MASWE-0117.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Inadequate Permission Management
+id: MASWE-0117
+alias: inadequate-permission-management
+platform: ["android", "ios"]
+profiles: ["P"]
+mappings:
+  masvs-v1: []
+  masvs-v2: [MASVS-PRIVACY-1]
+  cwe: [250]
+refs:
+- https://developer.apple.com/design/human-interface-guidelines/privacy#Requesting-permission
+- https://developer.apple.com/documentation/uikit/protecting_the_user_s_privacy/requesting_access_to_protected_resources
+- https://developer.android.com/training/permissions/requesting#explain
```

Comment:
Do we need these URLs with the anchor, as we anyway are referencing to the main page (https://developer.android.com/training/permissions/requesting)